apisix/plugin.lua

```diff
@@ -365,17 +365,7 @@ function _M.filter(ctx, conf, plugins, route_conf, runned_plugins)
         return plugins or core.tablepool.fetch("plugins", 0, 0)
     end
 
-    local unrunn_plugins
-    local runned_plugins_names
-    if runned_plugins and type(runned_plugins) == "table" then
-        runned_plugins_names = core.table.new(#runned_plugins / 2, 0)
-        for i = 1, #runned_plugins, 2 do
-            core.table.insert(runned_plugins_names, runned_plugins[i].name)
-        end
-        unrunn_plugins = core.table.new(4, 0)
-    end
-
-    local route_plugin_conf = route_conf and route_conf.value and route_conf.value.plugins
+    local route_plugin_conf = route_conf and route_conf.value.plugins
```

Comment:
```suggestion
    local route_plugin_conf = route_conf and route_conf.value and route_conf.value.plugins
```
Keeping the old way should be more rigorous.

Reply:
hi @shuaijinchao what you reviewed is already the previous version.